samples/DotNet/Microsoft.Azure.EventHubs/Rbac/AzureEventHubsSDK/README.md

```diff
@@ -1,3 +1,7 @@
 # Role based access sample with Microsoft.Azure.EventHubs SDK #
 
 For more information on Role based access (RBAC) and how to run this sample follow [this](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-role-based-access-control) link.
+
+**Note**: This sample uses the legacy Event Hubs library `Microsoft.Azure.EventHubs`. We strongly recommend you to use the new library `Azure.Messaging.EventHubs`. 
+
+Leveraging `Azure.Identity`, the new library simplifies the authentication workflow with `DefaultAzureCredential`. Under the hood, `DefaultAzureCredential` combines commonly used credentials, such as managed identity, authentication information set via environment variables or from a logged in Visual Studio Code Azure Account, to authenticate. It attempts to authenticate using the credentials in a specific order. See [here](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/identity/Azure.Identity#azure-identity-client-library-for-net) for more details.
```

Comment:
Did we run this by @jsquire? Perhaps he has some common text around DefaultAzureCredential we can re-use? I'd much rather point to a common place that talks about DefaultAzureCredential than explain its inner workings here

Reply:
I'm not sure that detail around the specifics of `Azure.Identity` makes a ton of sense here, as it's not germane to the sample.   I may suggest going with something that just extols the benefits.  As a rough suggestion, maybe something, like:

_"Authorization was one of the most consistent areas of feedback for the Azure SDKs from our customers.  Libraries for different Azure services have not had a consistent approach to authorization, roles, many developers have felt that the learning curve was difficult, and the APIs did not offer a good, approachable, and consistent story for integrating with Azure Active Directory principals._

_The [`Azure.Identity`](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/identity/Azure.Identity/README.md#azure-identity-client-library-for-net) library is intended to address that feedback and provide an approachable and flexible authentication experience across the Azure SDKs, including the `Azrue.Messaging.Eventhubs` library."_

Reply:
Yeah, my intent was to point out some benefits of how Track 2 does auth so people would actually click on the link :D I think what Jesse has makes sense here.

Reply:
Tweaking Jesse's comment a bit: 

Authorization was one of the most consistent areas of feedback for the Azure SDKs from our customers. Libraries for different Azure services did not had a consistent approach to authorization, roles, etc. The APIs also did not offer a good, approachable, and consistent story for integrating with Azure Active Directory principals. As such, many developers felt that the learning curve was difficult.

The [Azure.Identity](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/identity/Azure.Identity#azure-identity-client-library-for-net) library is intended to address that feedback and provide an approachable and flexible authentication experience across the Azure SDKs, including the `Azure.Messaging.EventHubs` library.